src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/NewAzureStorageAccount.cs

```diff
@@ -134,6 +134,40 @@ public PSNetworkRuleSet NetworkRuleSet
             get; set;
         }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Enable Azure Files AAD Integration for the Storage account.")]
+        [ValidateNotNullOrEmpty]
+        public bool EnableAzureFilesAadIntegration
```

Comment:
change the type to a SwitchParameter instead of a bool. See [here](https://github.com/Azure/azure-powershell/blob/bbe5ee957de07032bcf7f9062bba91cceee62565/src/ServiceManagement/Compute/Commands.ServiceManagement/Migration/MoveAzureNetworkResourceBase.cs#L32) for example

Reply:
Anyway, this will not be in Build, so I will remove it.